deploy/eck-stack/charts/eck-elasticsearch/values.yaml

```
@@ -267,7 +267,7 @@ nodeSets:
     # - command:
     #   - sh
     #   - "-c"
-    #   - bin/elasticsearch-plugin install --batch repository-s3
+    #   - bin/elasticsearch-plugin remove --purge repository-s3 ; bin/elasticsearch-plugin install --batch repository-s3
```

Let's not use a plugin that is already installed since ES 8.0 in this example.
```
plugin [repository-azure] is no longer a plugin but instead a module packaged with this distribution of Elasticsearch
-> removing [repository-azure]...
-> Installing repository-azure
[repository-azure] is no longer a plugin but instead a module packaged with this distribution of Elasticsearch
-> Please restart Elasticsearch to activate any plugins installed
```

```suggestion
    #   - bin/elasticsearch-plugin remove --purge analysis-icu ; bin/elasticsearch-plugin install --batch analysis-icu
```